prow/config.yaml

```diff
@@ -98,14 +98,29 @@ branch-protection:
         istio.io:
           branches: *blocked_branches
         operator:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
+            teams:
+              - repo-admins
           branches: *blocked_branches
         cni:
           branches: *blocked_branches
         installer:
           branches: *blocked_branches
         pkg:
+          required_pull_request_reviews: &tide_reviews
+            require_code_owner_reviews: true
+          restrictions: &tide_restrictions
```

Comment:
The tide_restrictions anchor should only be defined once. Thereafter we only need to point to it: `restrictions: *tide_restrictions` (the teams part is already included in the `*`).

If you need to extend the anchor then you need the
```yaml
restrictions:
  <<: *tide_restrictions
  my-extensions: whatever
```
pattern, but you don't here, so just `restrictions: *tide_restrictions` is right